src/seed/stub/ts.stub

```diff
@@ -0,0 +1,14 @@
+import * as Knex from "knex";
+
+exports.seed = function (knex: Knex): Promise<any> {
+    // Deletes ALL existing entries
+    return knex("table_name").del()
+        .then(function () {
+            return Promise.all([
```

Comment:
Promise.all should not be promoted here, since people are using it wrong in places where order of queries matters. It should be fixed in every other stub as well.